src/restconfig/src/main/java/org/geoserver/rest/catalog/LayerGroupController.java

```diff
@@ -198,6 +203,11 @@ public boolean supports(MethodParameter methodParameter, Type targetType, Class<
 
     @Override
     public void configurePersister(XStreamPersister persister, XStreamMessageConverter converter) {
+        // add keyword object configuration
+        XStream xstream = persister.getXStream();
```

Comment:
The same converter is used and he correctly encodes the keywords content is just the tag of the element that is not like ``<string>``. Anyway I'm explicitly forcing the tag to be ``<string>``.